include/simdjson/generic/ondemand/object.h

```diff
@@ -11,6 +11,23 @@ namespace simdjson {
 namespace SIMDJSON_IMPLEMENTATION {
 namespace ondemand {
 
+#if defined(__cpp_concepts) && defined(__cpp_consteval)
+#define SIMDJSON_SUPPORTS_EXTRACT 1
```

Comment:
Should this be enabled only when `SIMDJSON_EXCEPTIONS` is true? Systems such as Node.js use simdjson, they compile with C++20, but they disable exceptions (at the compiler level). All of the simdjson library is designed to work well with both exceptions and no exception. It would be ok to have features requiring exception support, although this should be explicit.

For reference, here is how it uses in Node.js (without exceptions) 
https://github.com/nodejs/node/blob/b887942e6b430e5f28ea7cb6c43841730161641c/src/node_modules.cc#L137

Reply:
We wouldn't need to specify that. Other than the one that you mentioned above (which is an easy fix), in `extract`, any type of error should be silently ignored; all the `noexcept` business is for the user's own exceptions (inside lambdas for example) not invalid json inputs.